### PR TITLE
[New] Gameforge Client

### PR DESCRIPTION
**Pull request by Dansito**
1) The client is made to work automatically when installed, so the installation never ends in bottles, the launcher installs properly and works fine but bottles expects to finish a process that never ends, the documentation does not specify an argument for this case.

2)The client has a taskbar indicator which integrates well into at least (kde) so it should work fine on any desktop with appindicator support.

3) I still have to test the installation and how the games are going, due to my limited connection I could not complete this job.

4) If the installer passes the maintainers' tests, it is necessary to move the .exe to another address, it is in a test repository (mine), if it can be moved to the site where the other installers are hosted, such as battle.net and epic store, it would be perfect.

### CODE REVIEW

File: Games/gameforgeclient.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- d3dcompiler_47
+
+Parameters:
+  dxvk: true
+  sync: esync
+  discrete_gpu: true
+
+Executable:
+  name: GameforgeInstaller
+  icon: gameforge.ico
```
Comment (mirkobrombin) [inline Games/gameforgeclient.yml:20]: please use a svg or png icon instead

**Review by mirkobrombin — CHANGES_REQUESTED** [review]

**Comment by mirkobrombin** [comment]
Tested Metin2, working fine.
image

image

image

File: Games/gameforgeclient.yml (hunk 1 above)
Comment (mirkobrombin) [reply Games/gameforgeclient.yml:20]: also add screenshot.png in the data folder please

Comment (Dansito) [reply Games/gameforgeclient.yml]: Done!

**Comment by Dansito** [comment]
It only remains to fix the issue of the completion of the installation, I suppose you already know a solution?
When you know the solutions the client will be ready for the merger

**Comment by Dansito** [comment]
I still haven't been able to solve the problem that the installation in bottles doesn't finish. I already use monitoring and check, any other solution? :(

**Pull request merged**